### PR TITLE
Update readme.md links to new site

### PR DESCRIPTION
**Pull request by Daeraxa**
Links lead to the repo docs, now the new site is live we can redirect the buttons to the general docs section, the installation section (which is yet to be completed) and the the build instructions.

This will clash with #81 and #74 (if the latter is to be merged) but both are easy conflicts to resolve.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,11 +43,11 @@
 
 <!---------------------------{ Files }--------------------------->
```
Comment (kaosine) [inline README.md:44]: Probably should change this line but otherwise looks good!

**Review by kaosine — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kaosine) [reply README.md:44]: @Daeraxa update on this, maybe?

Comment (Daeraxa) [reply README.md:44]: Sorry missed this. What, just rename it to links or something?

Comment (kaosine) [reply README.md:44]: Yea, links would be the ideal one yes

Comment (Daeraxa) [reply README.md]: Links was already being used so renamed to Documents which can be for both website links and repo files if necessary.

File: README.md [hunk 2]
```diff
@@ -41,13 +41,13 @@
...
 [#]: #
 
-<!---------------------------{ Files }--------------------------->
+<!---------------------------{ Documents }--------------------------->
 
-[Documentation]: docs/Documentation.md 'Information how to use & work with Pulsar.'
-[Install]: docs/Installation.md 'How to install Pulsar on your system.'
+[Documentation]: https://pulsar-edit.dev/docs/ 'Information how to use & work with Pulsar.'
+[Install]: https://pulsar-edit.dev/docs/launch-manual/sections/getting-started/#installing-pulsar 'How to install Pulsar on your system.'
 [Retired]: docs/Retirement.md 'Check out what happened to the original Atom project.'
```
Comment (Spiker985) [inline README.md:48]:
```suggestion
[Retired]: https://github.blog/2022-06-08-sunsetting-atom/ 'Check out what happened to the original Atom project.'
```

**Review by Spiker985 — CHANGES_REQUESTED** [review]
Added link to GitHub sunset blog post from https://github.com/pulsar-edit/pulsar/pull/108/

**Comment by Spiker985** [comment]
This should also close #64 when merged

**Review by Spiker985 — APPROVED** [review]

**Comment by Spiker985** [comment]
@kaosine Since I don't have write access, I can't approve the PR to allow it to show it's merge status. You mind taking a look, and hopefully approving?

**Comment by Daeraxa** [comment]
I sync'd the branch to include your commit from #81 so that shouldn't cause a merge issue now, it was still pending when I made this PR.

**Review by confused-Techie — APPROVED** [review]
Since I managed to miss it the first time, here's my approval.

Looks great, and I'll go ahead and remove @kaosine's changes since they seem to be addressed, and you are wanting to check the merge status.

**Comment by confused-Techie** [comment]
Alright, this PR has been active for over a month.

Since I've already approved it, and it is a non-code changing PR, meaning we only need one approval, I'll go ahead and merge.

Thanks for the contribution!

**Pull request merged**